nomad/dns.nomad

```diff
@@ -0,0 +1,59 @@
+job "dns" {
+  datacenters = ["dc1"]
+  type        = "system"
+
+  group "dnsmasq" {
+    network {
+      mode = "bridge"
+      port "dns" {
+        static = 53
+        to     = 53
+      }
+    }
+
+
+    task "dnsmasq" {
+      driver = "docker"
+
+      config {
+        #This is an alpine-based dnsmasq container
+        image = "4km3/dnsmasq:2.85-r2"
+        ports = ["dns"]
+        args = [
+          # Send all queries for .consul to the NOMAD_IP
+          "--server", "/consul/${NOMAD_IP_dns}#8600",
```

Comment:
Note to self: doublecheck why we're doing it this way. It probably makes sense to switch to `attr.unique.network.ip-address`

Reply:
(how likely are we to actually follow up on this lol)

Reply:
This part seems like it will work both ways, so I'm going to leave it